cluster/ci/monitoring/mixins/config.libsonnet

```diff
@@ -21,6 +21,11 @@
         channel: '#build-cop-alerts',
         notify: 'build-cop',
       },
+      'openshift-library': {
```

Comment:
i'm not sure i understand the api...   does this mean we'll get alerts on any failure for jobs run against the openshift-library repo?  Can we not specify specific jobs?  Also will this apply to PR jobs or only periodic jobs?

Reply:
No. `'openshift-library'` is just the name of receiver. Its usage is explained in https://github.com/openshift/release/pull/5341#issuecomment-539661669

Reply:
got it, thanks.